scripts/datenlord-monitor-deploy.sh

```diff
@@ -4,29 +4,45 @@ set -o errexit
 set -o nounset
 set -o xtrace
 
-if [ "$1" = "helm" ]
+if [ $# -le 0 ]
 then
-    kubectl create namespace datenlord-monitoring 
+    cat <<EOF
+Usage: $0 [deploy|undeploy|helm]
+
+deploy      Deploy datenlord monitor by yaml file.
+helm        Deploy datenlord monitor by helm.
+undeploy    Undeploy all datenlord monitor.
+EOF
+    exit 1
+fi
+
+if [ ${1-default} = "helm" ]
```

Comment:
we don't need default if we force user to give one parameter.

Reply:
Done